docset/windows/storage/Format-Volume.md

```diff
@@ -82,19 +82,40 @@ To create a new volume, use this cmdlet in conjunction with the Initialize-Disk
 ## EXAMPLES
 
 ### Example 1: Quick format
-```
+```powershell
 PS C:\>Format-Volume -DriveLetter D
 ```
 
 This example performs a format of the D volume.
 
 ### Example 2: Full format using FAT32
-```
+```powershell
 PS C:\>Format-Volume -DriveLetter D -FileSystem FAT32 -Full -Force
 ```
 
 This example performs a full format of the D volume using the FAT32 file system.
 
+### Example 3: Format all D drives across a cluster
+```powershell
+PS C:\> Get-Volume -DriveLetter D
+
+DriveLetter FileSystemLabel FileSystem DriveType HealthStatus OperationalStatus SizeRemaining      Size
+----------- --------------- ---------- --------- ------------ ----------------- -------------      ----
+D           Server1         NTFS       Fixed     Healthy      OK                    126.76 GB 126.87 GB
+D           Server2         NTFS       Fixed     Healthy      OK                    126.76 GB 126.87 GB
+
+
+PS C:\> Format-Volume -DriveLetter D
+
+DriveLetter FileSystemLabel FileSystem DriveType HealthStatus OperationalStatus SizeRemaining      Size
+----------- --------------- ---------- --------- ------------ ----------------- -------------      ----
+D                           NTFS       Fixed     Healthy      OK                    126.76 GB 126.87 GB
+D                           NTFS       Fixed     Healthy      OK                    126.76 GB 126.87 GB
+
+```
+
+This example gets all drives with letter D on the cluster and then formats them. This would format all D drives without confirmation!
```

Comment:
Can we use a more formal sentence without the exclamation mark here to warn the users? 
for example _Be careful, if using this cmdlet on a Windows Cluster, it would format all drives returned by the Get-Volume_, something like that.